docs/sources/installation/docker.md

```diff
@@ -81,7 +81,7 @@ docker run -d \
 
 > Only available in Grafana v5.3.1 and later.
 
-You can install plugins from custom URLs by specifying the URL like this: `GF_INSTALL_PLUGINS=<url to plugin zip>;<plugin name>`.
+You can install plugins from custom URLs by specifying the URL like this: `GF_INSTALL_PLUGINS=<url to plugin zip>;<plugin install folder name>`.
```

Comment:
Why the change from "plugin name" to "plugin install folder name"? Doesn't make sense to me, based on my knowledge of the process, but I could be missing something.

Reply:
Yeah absolutely is confusing. They become fundamentally different things depending on the `pluginUrl` argument being supplied or not

Reply:
I think it's worth having a discussion about refactoring these to make it easier to reason about

Reply:
I think it would be better to solve that problem of grafana-cli having a confusing interface separately. Maybe it would be better to get rid of `--pluginUrl` and download from a URL if that is provided instead of an ID. The folder name would then be surmised in the same way as when downloading the plugin from grafana.com.

Reply:
Any reason why this option was given @marefr? Or was it just an oversight when extending the command to also fetch via URL?

Reply:
> Why the change from "plugin name" to "plugin install folder name"?

Agree, that sounds wrong

> Any reason why this option was given @marefr? Or was it just an oversight when extending the command to also fetch via URL?

Not sure which option you're referring to. It accepts a combination of URL's and plugin names. The URL is to allow custom plugins to be installed for those not having their plugin published on grafana.com

Reply:
Sorry, now I understand. The example is for one plugin, thought it was for 2. Then it makes sense I think

Reply:
However, the example could be extended to include an additional plugin

```suggestion
You can install plugins from custom URLs by specifying the URL like this: `GF_INSTALL_PLUGINS=<url to plugin zip>;<plugin install folder name>,grafana-clock-panel`.
```

Reply:
Please put this in singular tense to eliminate that confusion.

Reply:
Yeah the main confusion is this:

Installing an internal plugin (IE lives on grafana.com):
```
grafana-cli plugins install test-plugin
```
Which downloads the zip associated with `test-plugin` from grafana.com and extracts the contents into a folder named `test-plugin`.

Installing an plugin externally:
```
grafana-cli --pluginUrl="https://storage.google.com/plugins/zabbix-app-1.0.zip" plugins install zabbix-plugin
```
Which downloads the zip and extracts the contents to `zabbix-plugin` or whatever folder name we care to specify.

Maybe we can deprecate `pluginUrl` and refactor to support both like so?
```
grafana-cli plugins install zabbix-plugin 1.0
```

```
grafana-cli plugins install https://storage.google.com/plugins/zabbix-app-1.0.zip
```

Reply:
> Maybe we can deprecate pluginUrl and refactor to support both like so?

Maybe, but don't see any point of doing that in this PR.